tests/amqp_frame_codec_ut/amqp_frame_codec_ut.c

```diff
@@ -993,7 +993,7 @@ TEST_FUNCTION(amqp_performatives_are_encoded_successfully)
 
         // assert
         ASSERT_ARE_EQUAL(int, 0, result);
-        (void)sprintf(test_string, "test %zu", i);
+        (void)sprintf(test_string, "test %lu%lu", i);
```

Comment:
>i [](start = 50, length = 1)

Could you please cast this to (unsigned long) to be explicit about it?
Thanks!

Reply:
![zu](https://user-images.githubusercontent.com/16568758/45006078-b3b60b80-b029-11e8-975c-6886527cbbd8.png)
when using %zu, message cannot be shown in correct format on the platform of mbed os5. As I modified it to %lu, the message can correctly shown as below.
![lu](https://user-images.githubusercontent.com/16568758/45006117-dfd18c80-b029-11e8-9ecb-81648f02a65d.png)
And sorry about the "%lu%lu" mistake, I have corrected it and pushed my modification.

Best Regards